### PR TITLE
chore: reorganize exports

### PR DESCRIPTION
**Pull request by jsomsanith-tlnd**
**What is the problem this PR is trying to solve?**

We started to do UMD builds and we decided not to embed react-bootstrap in @talend/react-components.
We attached every component related to the target component to expose everything in a logical manner.

**What is the chosen solution to this problem?**

- In @talend/react-components remove react-bootstrap exports.
- Remove extra exports.
- attache link to umd in surge

**Please check if the PR fulfills these requirements**

* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

**Review by jmfrancois — APPROVED** [review]

File: packages/containers/package.json [hunk 1]
```diff
@@ -74,11 +74,11 @@
...
-    "@talend/icons": "^5.0.0",
-    "@talend/react-cmf": "^5.0.0",
+    "@talend/icons": "^6.0.0-alpha.10",
+    "@talend/react-cmf": "^6.0.0-alpha.10",
     "@talend/react-cmf-router": "^3.5.0",
-    "@talend/react-components": "^5.0.0",
-    "@talend/react-forms": "^5.0.0",
+    "@talend/react-components": "^6.0.0-alpha.10",
+    "@talend/react-forms": "^6.0.0-alpha.10",
     "i18next": "^15.1.3",
```
Comment (frassinier) [inline packages/containers/package.json:82]: We can align right here I guess

File: packages/datagrid/package.json [hunk 2]
```diff
@@ -63,9 +63,9 @@
     "style-loader": "^0.23.0"
   },
   "peerDependencies": {
-    "@talend/icons": "^5.26.0",
-    "@talend/react-cmf": "^5.26.0",
-    "@talend/react-components": "^5.26.0",
+    "@talend/icons": "^6.0.0-alpha.41",
+    "@talend/react-cmf": "^6.0.0-alpha.41",
+    "@talend/react-components": "^6.0.0-alpha.41",
```
Comment (frassinier) [inline packages/datagrid/package.json:68]: We can align with version.js cmd

File: packages/forms/package.json [hunk 3]
```diff
@@ -76,7 +76,7 @@
     "storybook-addon-i18next": "^1.2.1"
   },
   "peerDependencies": {
-    "@talend/react-components": "^5.0.0",
+    "@talend/react-components": "^6.0.0-alpha.41",
```
Comment (frassinier) [inline packages/forms/package.json:79]: We can align

File: packages/stepper/package.json [hunk 4]
```diff
@@ -63,8 +63,8 @@
     "style-loader": "^0.23.0"
   },
   "peerDependencies": {
-    "@talend/icons": "^5.0.0",
-    "@talend/react-components": "^5.0.0",
+    "@talend/icons": "^6.0.0-alpha.41",
+    "@talend/react-components": "^6.0.0-alpha.41",
```
Comment (frassinier) [inline packages/stepper/package.json:67]: We can align

File: packages/storybook-cmf/package.json [hunk 5]
```diff
@@ -44,7 +44,7 @@
   },
   "peerDependencies": {
     "@storybook/react": "^5.3.1",
-    "@talend/react-cmf": "^5.0.0",
+    "@talend/react-cmf": "^6.0.0-alpha.41",
```
Comment (frassinier) [inline packages/storybook-cmf/package.json:47]: We can align

File: versions/dependencies.json [hunk 6]
```diff
@@ -9,7 +9,7 @@
   "whatwg-fetch": "^2.0.3",
   "//comment_ui_scripts": "// UI deps outside of the release life cycle",
   "@talend/scripts": "^0.16.1",
-  "@talend/react-cmf-router": "^3.2.1",
+  "@talend/react-cmf-router": "^3.4.0-alpha-5",
```
Comment (frassinier) [inline versions/dependencies.json:12]: oh no!

File: packages/components/src/InlineMessage/InlineMessage.component.js [hunk 7]
```diff
@@ -21,7 +21,6 @@ import { getTheme } from '../theme';
 	};
  <InlineMessage {...props}/>
  */
-// TODO 6.0: do not export this
 export const TYPES = {
```
Comment (frassinier) [inline packages/components/src/InlineMessage/InlineMessage.component.js:24]: export is still there?

**Review by frassinier — APPROVED** [review]

**Pull request merged**